package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -629,6 +630,10 @@ export const MessageInputProvider = ({
             message,
             options: sendOptions,
           });
+          // Even though we edit, but we eventually send the message as a regular message, so we need to clear the editing state.
+          if (editedMessage) {
+            clearEditingState();
```

Comment:
if we're sure about doing this, we probably want to do it before sending the message itself, just like we do for the other bits of clearing the composer.

Reply:
done